mne/viz/ica.py

```diff
@@ -588,8 +610,7 @@ def plot_ica_overlay(ica, inst, exclude=None, picks=None, start=None,
     from ..evoked import Evoked
     from ..preprocessing.ica import _check_start_stop
 
-    if not isinstance(inst, (BaseRaw, Evoked)):
-        raise ValueError('Data input must be of Raw or Evoked type')
+    _validate_type("inst", inst, "Raw or Evoked", (BaseRaw, Evoked))
```

Comment:
This one for example probably should still say `Raw or Evoked` so this would be one of the few non-automatic ones